src/improvements/tasks/sep/033-arena-z-u15-to-u16a/config.toml

```diff
@@ -0,0 +1,28 @@
+l2chains = [
+    {name = "arena-z-testnet", chainId = 9899}
+]
+
+templateName = "OPCMUpgradeV410"
+
+[[opcmUpgrades]]
+chainId = 9899
+absolutePrestate = "0x039fb00f964358d326de5804afb4e70a1ca34f9e56c60350b6c8a41fff302490"
```

Comment:
TODO: Before merging we should make sure with proofs that this prestate is correct, can't find out where it's come from.